apps/cowswap-frontend/src/cowEventEmitter.ts

```diff
@@ -0,0 +1,3 @@
+import { CowEventEmitterImpl } from '@cowprotocol/events'
+
+export const cowEventEmitter = new CowEventEmitterImpl()
```

Comment:
Cool! Now we have it as a global const, do we still need `CowEventEmitterContext`?

Reply:
Hehe, technically not. I just thought its nicer to depend in react on the context for emitting in components. The context exposes the interface only, not the implementation. 

TBH, I think is not great the global var, I was not planning to do it, but I also hated the idea of adding it in the redux action payload. So you would prefer to kill the context?

Reply:
https://github.com/cowprotocol/cowswap/pull/3839